libpldmresponder/bios_config.cpp

```diff
@@ -128,11 +128,15 @@ void BIOSConfig::initBIOSAttributes(const std::string& systemType,
 
     buildTables();
 
+    info("Register listenPendingAttributes");
+    listenPendingAttributes();
+
     if (pendingAttributes.size())
     {
         try
         {
-            info("Setting Pending Attributes");
+            info("Setting Pending Attributes {NUMBER}", "NUMBER",
+                 (int)pendingAttributes.size());
```

Comment:
can't we print the attribute names here ? size might not give much clues if we hit this path.